PathPlanning/BreadthFirstSearch/bfs.py

```diff
@@ -0,0 +1,270 @@
+"""
```

Comment:
The file name bfs.py -> breadth_first_search.py is better

Reply:
Done.